package.json

```diff
@@ -11,7 +11,7 @@
   ],
   "scripts": {
     "build": "turbo run build --scope=\"@calcom/web\" --include-dependencies",
-    "clean": "turbo run clean && rm -rf node_modules",
+    "clean": "find . -name node_modules -o -name .next -o -name .turbo -o -name dist -type d -prune | xargs rm -rf",
```

Comment:
Nice. Didn't know you can do OR like this 👍

Reply:
Doesn't handle special cases like this. Not a blocker but why are we moving away from the earlier approach.
Maybe we can use both, for cases like this.
<img width="1001" alt="Screenshot 2022-07-22 at 7 36 07 PM" src="https://user-images.githubusercontent.com/1780212/180456382-244f91ae-576c-45b1-a696-b2a84dc25d83.png">
)

Reply:
Will follow up @hariombalhara